src/local-development/code-pull-deploy.ts

```diff
@@ -25,12 +26,12 @@ import { sendTelemetry } from '../utils/telemetry';
  *   - 	For module: api, parameteres, expect, interface, samples, scope
  */
 export async function pullComponentCode({
-	appComponentType,
-	remoteComponentName,
-	codeType,
-	origin,
-	destinationPath,
-}: {
+											appComponentType,
+											remoteComponentName,
+											codeType,
+											origin,
+											destinationPath,
+										}: {
 	appComponentType: AppComponentType | AppGeneralType;
```

Comment:
Your change reformatted the function parameters with non-standard indent.... This is not the prettier format, correct? :)

Reply:
yes, I have prettier onSave, always changed to this style...

Reply:
Outdated, resolved.